src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -475,15 +476,21 @@ public HttpResponse doTerm() {
             @Override public void onFailure(@NonNull Throwable t) {}
         });
         printLater(StopState.KILL, "Click here to forcibly kill entire build");
+    }
+
+    /** Sends {@link StepContext#onFailure} to all running (leaf) steps. */
+    @RequirePOST
+    @WebMethod(name = { "term" })
```

Comment:
```suggestion
    @WebMethod(name = "term")
```
conventionally

Reply:
probably also want to mark `DoNotUse` to make it clear this is here only for use from Stapler